doc/command_ref.rst

```diff
@@ -174,22 +174,22 @@ Options
     Disable specific repositories by an id or a glob. This option is mutually exclusive with ``--repo``.
 
 ``--downloaddir=<path>``
-    Redirect downloaded packages to provided directory. The option has to by used together with \-\
-    :ref:`-downloadonly <downloadonly-label>` command line option or with
-    ``download`` command (dnf-plugins-core) or with ``system-upgrade`` command
+    Redirect downloaded packages to provided directory. The option has to be used together with the \-\
```

Comment:
```suggestion
    Redirect downloaded packages to the provided directory. The option has to be used together with the \-\
```

Reply:
Hmm, does the suggestion create a separate commit? I'd rather fixup the original one.